src/tools/autosave.js

```diff
@@ -58,13 +60,13 @@ function updateAutosave(state, original) {
 
 let autoSaveTimeout
 
-function subscribeToAutoSave(store) {
+async function subscribeToAutoSave(store) {
```

Comment:
Does this function actually need to be async?

Reply:
No